src/seedu/addressbook/commands/AddCommand.java

```diff
@@ -60,6 +60,7 @@ public ReadOnlyPerson getPerson() {
     @Override
     public CommandResult execute() {
         try {
+            toAdd.assignSequenceNumber();
```

Comment:
recollect the discussion during the tutorial on moving this line after the `addPerson` method. think why that may be a better choice.